### PR TITLE
Refactor specifiers to components naming

### PR DESCRIPTION
**Pull request by MDUYN**

### CODE REVIEW

**Comment by MDUYN** [comment]
@fizaashraf37 I made some refactors to reflect all changes and fix some tests

**Review by fizaashraf37 — COMMENTED** [review]
@MDUYN  I have reviewed the whole refactored code. The code structure and changes look good. I have added few suggestions in some files. You can make changes accordingly

File: flask_swagger_generator/utils/url_variable_type.py [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+    @staticmethod
+    def from_string(value: str):
+
+        if isinstance(value, str):
+
+            if value.lower() == 'string':
+                return UrlVariableType.STRING
+            elif value.lower() == 'int':
+                return UrlVariableType.INTEGER
+            elif value.lower() == 'float':
```
Comment (fizaashraf37) [inline flask_swagger_generator/utils/url_variable_type.py:21]: Float, path and uuid types are not recognized by swagger although these can be added in the url. So these should not be used to resolve input type in SwaggerPathParameter class. We can add some mapping for url variable type to swagger path types. 
The only allowed values by swagger are: **array, boolean, integer, number, object, string**

We can use these mappings:
```
float : number
path : string
uuid : string (not sure about this one, I will need to look into this)
```

**Review by fizaashraf37 — CHANGES_REQUESTED** [review]
Need to update the URL_Variable_Type class as these input types are not supported in swagger. Might need to use a mapper for these.

File: flask_swagger_generator/specifiers/version_three_specifier.py [hunk 2]
```diff
@@ -0,0 +1,319 @@
...
+import re
+from datetime import datetime
+from typing import List
+
+from marshmallow import Schema as MarshmallowSchema
+
+from flask_swagger_generator.components.swagger_models import SwaggerModel
+from flask_swagger_generator.specifiers.swagger_specifier \
+    import SwaggerSpecifier
+from flask_swagger_generator.components.version_three import \
```
[the remark this answers is not in the document]
Comment (MDUYN) [reply flask_swagger_generator/specifiers/version_three_specifier.py:11]: Good point, thanks

File: flask_swagger_generator/utils/url_variable_type.py (hunk 1 above)
Comment (MDUYN) [reply flask_swagger_generator/utils/url_variable_type.py:21]: I removed the references to uuid, path and float

```python
def get_flask_url_variable_type_value(self):

    if self == UrlVariableType.STRING:
        return 'string'
    elif self == UrlVariableType.INTEGER:
        return 'integer'
    elif self == UrlVariableType.FLOAT:
        return 'number'
    elif self == UrlVariableType.PATH:
        return 'string'
    elif self == UrlVariableType.UUID:
        return 'string'
```

Comment (MDUYN) [reply flask_swagger_generator/utils/url_variable_type.py:21]: Could you review the changes again

**Review by fizaashraf37 — APPROVED** [review]
The changes look good. Thanks for incorporating all the requested changes.

**Pull request merged**